src/create-release.js

```diff
@@ -26,6 +27,7 @@ async function run() {
       repo,
       tag_name: tag,
       name: releaseName,
+      body,
```

Comment:
That is a clever feature, and I could see how it would be handy. But feels like it is out of the scope for this pull request. Mostly because the test coverage and error handling for this would be more involved. 

Would you like to propose it as a follow up to these changes? I have a little concern about a magic string for the body, but will defer to @IAmHughes.

Reply:
I can open a PR against `csexton:add-body` (in your fork). When merged, this PR will be automatically updated.

Regarding the magic string, it's only a placeholder. I think it'd be better to use something like 'body: !my_filename.md', so that quotes are not required.

Reply:
I'm personally not a fan of a magic string prefix either, since it makes assumptions about the actual content of the body.

If anything, I'd prefer having a separate param like `body_from_file` (or whatever) and having it error if both are present, however I also feel like it could be handled as a separate PR.